rollup.config.js

```diff
@@ -37,8 +37,8 @@ export const appConfiguration = {
     }),
 
     svelte({
-      dev: !production,
       preprocess: sveltePreprocess(),
+      emitCss: false,
```

Comment:
Curious as to what this flag does?

Reply:
It's documented here: https://github.com/sveltejs/rollup-plugin-svelte#extracting-css

However, the build was failing because Rollup expected yet _another_ plugin to process the CSS. We have no such plugin, so I disabled this option!